tests/tools.vitruv.applications.pcmjava.tests/src/tools/vitruv/applications/pcmjava/tests/pcm2java/javahelper/JavaCreationToFieldAssignmentHelper.xtend

```diff
@@ -0,0 +1,61 @@
+package tools.vitruv.applications.pcmjava.tests.pcm2java.javahelper
+
+import org.emftext.language.java.expressions.ExpressionsFactory
+import org.emftext.language.java.instantiations.InstantiationsFactory
+import org.emftext.language.java.literals.LiteralsFactory
+import org.emftext.language.java.members.Field
+import org.emftext.language.java.operators.OperatorsFactory
+import org.emftext.language.java.references.ReferencesFactory
+import org.emftext.language.java.statements.Statement
+import org.emftext.language.java.statements.StatementsFactory
+import org.emftext.language.java.types.TypeReference
+
+enum ConstructorArguments {
+	WITH_NULL_LITERAL,
+	WITHOUT_NULL_LITERAL
+}
+
+class JavaCreationToFieldAssignmentHelper {
```

Comment:
all those helper classes should have visibility `package` (Xtend default is `public`)